test/e2e/lib/creation.go

```diff
@@ -0,0 +1,117 @@
+/*
+Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package lib
+
+import (
+	batchv1 "k8s.io/api/batch/v1"
+	"k8s.io/apimachinery/pkg/api/meta"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"knative.dev/eventing/test/lib/resources"
+
+	eventsv1alpha1 "github.com/google/knative-gcp/pkg/apis/events/v1alpha1"
+	messagingv1alpha1 "github.com/google/knative-gcp/pkg/apis/messaging/v1alpha1"
+	pubsubv1alpha1 "github.com/google/knative-gcp/pkg/apis/pubsub/v1alpha1"
+)
+
+func (c *Client) CreateUnstructuredObjOrFail(spec *unstructured.Unstructured) {
+	namespace := c.Namespace
+	gvr, _ := meta.UnsafeGuessKindToResource(spec.GroupVersionKind())
+	_, err := c.Core.Dynamic.Resource(gvr).Namespace(namespace).Create(spec, v1.CreateOptions{})
```

Comment:
nit: just use c.Namespace and don't need for that auxiliary namespace variable? Same as below

Reply:
Done.